recipes/echidna/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "echidna" %}
+{% set pname = "sc_echidna" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ pname[0] }}/{{ pname }}/{{ pname }}-{{ version }}.tar.gz"
+  sha256: 0e355842d01899c868de50b3a61356f22e5edfac22b57cdbe1d388204c2e2065 
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
```

Comment:
Are you sure about `"x.x"` ?
Version `1.x` suggests `"x"` if you follow SemVer.